Create 2026-01-07.md (#757)
solid/specification: meetings/2026-01-07.md

* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
* If this is your first time, welcome! Please introduce yourself.

---

## Introductions

## Actions Review

## Topics

### Local AI scribing

eP: tried to build something last year, but that only works on mac and windows, need to check that out again

### Chair election

https://github.com/w3c-cg/solid/blob/main/elections/2025/elected-candidates.txt

* eP: congratulations to Christoph and Theo 
* ...: please check if you have moderator access to https://github.com/w3c-cg/solid/settings
* ...: votes were anonymized by W3C staff
* ...: I want to encourage all to continue to work on what they want to get done (regardless of who is chair)
* CB: Shout out to Rui and Precious for joining. Also thanks to Theo.
* ...: On your question about w3c-cg/solid — I don't have moderator access at this moment
* eP: I will contact Ian to update roles

> * CB: We had some brainstorming about client cilent work and ... Please make yourself heard and we will do our best to support you.

Reviewer: ```suggestion
* CB: We had some discussion about client-to-cilent specifications on the mailing list during the election which was useful, I think. Please make yourself heard and we will do our best to support you.
```

* CB: First I would like to think how we approach this. To get people to share what they work on, people work on their usual sutuff. It comes to visibility in the group of who does what. Maintianing NSS for long time, over time you get to know the names. I was wondering if they could be a venue to get a first picture of what people in the community are working on. For example people can have a poster of step by, even if not there is a low barier option to have canvas where people can share what they do.
* ...: I would look over to Jesse if ODI has something planned or done. 
* JW: There is a catalog that Jeff works on, with some contributions from elf. It doesn't have a huge amount of what people work on. We could make it a source of truth. ODI is trying to start contribute to the catalog. 
* ...: I also have on the radar that ODI needs to provide meta tooling.
* eP: In paralel to catalog I've been also working on https://elf-pavlik.github.io/solid-efforts/#/
* ...: We made an attempt with Jeff to reconcile the work but it seems that we may target different audiences.

> * CB: Can we use mailing list or chat room to share what people are working on.

Reviewer 2: ```suggestion
* CB: Can we use mailing list or chat room to share what people are working on?
```

* JW: On the forum there was an introducion thread that doesn't seem to be used. I think mailing list is a good idea. In general our communication speces could be better. Is it the time to use something more accessible to the wider community like Discord. If you want ODI to provide it we could look into that.
* CB: Discord is a double edge sword for me, some people seem not to like it. I don't want to exclude them by imposing a choice of effectively a product. 
* eP: If we want to have some form a visibility or discovery, I don't think that a forum or mailing list is the best choice - I think that a structured knowledge base is the way to go, so people can browse what is there. Then, I see this complementary (a) people need to share it and also (b) people need to be able to find it way later.
* ...: re Discord, some people do not like closed source products - I used to be that way. Nowaday's I am a bit more open to using whatever tool that helps us move forward.
* Michal: I am curious to hear why Discord is better than Matrix.

> * eP: Many communities use Discord, e.g. the local-first communities seem to use Discord. From a UX perspective, Matrix is a mess for me (push notifications on mobile when chatting on desktop). Jesse, why did you propose Discord?

Reviewer 2: ```suggestion
* eP: Many communities use Discord, e.g., the local-first communities seem to use Discord. From a UX perspective, Matrix is a mess for me (push notifications on mobile when chatting on desktop). Jesse, why did you propose Discord?
```

* JW: Meet the developers where they are at. Also, bridging seems to be reasonable. When I use matrix, I actually use a slack channel that is bridged to matrix. For the knowledge base, one project we could tackle would be setting up a pipeline to autmoatically extract structured data from e.g. the mailing list or matrix messages.
* ...: If the community desires changing communication infrastructure, I can help from the ODI's side.

> * eP: I think we could have a frist small vote to see what people think in this room. Maybe we could have a broader feedback round with the community via the email list.

Reviewer 2: ```suggestion
* eP: I think we could first have a small vote to see what people think in this room. Maybe we could have a round of broader feedback from the community via the email list.
```

> * th: I think it is alrady a bit complicated to communicate in the community - adding a new one could just make it worse. There are problems with Matrix but it is doing the basic thing necessary. I dont think this is the biggest issue we have right now.

Reviewer 2: ```suggestion
* th: I think it is already a bit complicated to communicate in the community. Adding a new tool could just make it worse. There are problems with Matrix, but it is doing the basic thing necessary. I don't think this is the biggest issue we have right now.
```

* ...: also, let's not fragment of the different groups (CG, practitioners, WG) communicate.
* ...: let's here some quick +/-1 on this:

Quick Poll: Do we want to use something like Discord
* CB: -1 (on just adding it to the channels, rather first communicate with the CG participants on what they would prefer via mailing list (as the normative communication channel))
* eP: +0.5 - I'm good with anything that someone will be responsible for making work
* Theo: -1 , open to create new channel with other tools ( with potential matrix bridge ) , but I believe the cost of migration ( and all debate it might generate ) are not worth the benefits. I'd rather put effort on better definings channel of discution ( what topic belong where )
* Michal:  -1 on migrating to Discord, happy with someone implementing bridging as long as Matrix stays source of truth.
* Luke: -1 I'd rather encourage open solutions and potentially something more "dogfooding" to help people onboard to Solid itself
* Precious: I am good with discord, as i am more familiar with it than Matrix
* AB: -1 not happy to add new channels

> * RZ: 0. Fine with adding new channels as long as bridges are provided. I'm used to communities with multiple channels but all bridged together. Forum should be brought back to people's attention.

Reviewer 2: ```suggestion
* CB: -1 (on just adding it to the channels; rather, first communicate with the CG participants via mailing list (as the normative communication channel) about what they would prefer)
* eP: +0.5 - I'm good with anything that someone will be responsible for making work
* Theo: -1, open to create a new channel with other tools (with potential matrix bridge), but I believe the cost of migration (and all debate it might generate) are not worth the benefits. I'd rather put effort on better defining channels of discussion (what topics belong where)
* Michal:  -1 on migrating to Discord, happy with someone implementing bridging, as long as Matrix stays the source of truth.
* Luke: -1 I'd rather encourage open solutions and potentially something more "dogfooding" to help people onboard to Solid itself
* Precious: I am good with Discord, as I am more familiar with it than Matrix
* AB: -1 not happy to add new channels
* RZ: 0. Fine with adding new channels as long as bridges are provided. I'm used to communities with multiple channels but all bridged together. Forum should be brought back to people's attention.
```

> eP: let's come back to discussing what is up for the next months

Reviewer 2: ```suggestion
* eP: let's come back to discussing what is up for the next months
```

> * CB: I will be going back to the spec, knowing that WG is redefining a lot of things and taking different directions and keeping track of those differences. Where WG adds pieces and chooses to use different pieces. Identifying what gaps need to be filled. For example adding secion to Solid-OIDC on the issuer check, implementations came around to implementing that. Knowing that this is part of LWS protocol

Reviewer 2: ```suggestion
* CB: I will be going back to the spec, knowing that the WG is redefining a lot of things and taking different directions and keeping track of those differences where the WG adds pieces and chooses to use different pieces. Identifying what gaps need to be filled. For example adding section to Solid-OIDC on the issuer check, implementations came around to implementing that. Knowing that this is part of LWS protocol.
```

* eP: that would be a good idea to track the progress / what is happening in the WG
* JW: on the gap-filling part, I will work with aaron and laurens on access requests and grants. This is in parallel with LWS.
* eP: I have an implementation of access requests and grants in SAI. I know these grants are different from what Inrupt does.
* ...: Also, I am interested in how the type indexes are aligned between Solid and LWS.

> * ...: BTW, I talked with Erich for him to present the work on what is the latest state of CRUD in LWS. (Sorry for putting you on the spot) For example for next week?

Reviewer 2: ```suggestion
* eP: I have an implementation of access requests and grants in SAI. I know these grants are different from what Inrupt does.
* ...: Also, I am interested in how the type indexes are aligned between Solid and LWS.
* ...: BTW, I talked with Erich for him to present the work on the latest state of CRUD in LWS. (Sorry for putting you on the spot.) Maybe for next week?
```

* EB: Not the first and not the last time :) I'd be happy to.
* eP: I'll put it on the agenda for next week.
* CB: Jesse I would like on that with you, please invite me. I drafted a small demo for Authorization App github.com/uvdsl/solid-authorization-app

> * ...: It works based on data model that Pavlik works on. I have a lot of thougts to make it work with what we have without being to opinionated on it.

Reviewer 2: ```suggestion
* CB: Jesse, I would like to get in on that with you; please invite me. I drafted a small demo for _Authorization App_, https://github.com/uvdsl/solid-authorization-app
* ...: It works based on the data model that Pavlik is working on. I have a lot of thoughts to make it work with what we have without being to opinionated about it.
```

And, there is also the Solid-File-Manager.
* eP: Having a way to store all the history of prompts and chats in a storage would be usefull indeed. NLnet works on more transparency on LLM use it the project they fund. 
* PO: Still working on it, but there is no PR yet. Once ready, I'll share the link.
* eP: I think it would be great to have presentations in both practitioners (higher-level maybe?) and CG (in technical detail). So maybe, we could have a bi-weekly presentation of implementations and a corresponding deep dive. If we can really lean on the implementations then our work will be much more grounded rather than just drafting specs. My idea would be to have a schedule of demos, feedback and Q/A.
* ...: We are almost out of time ....
* JW: You talked about deliverables - what might be useful could be a little bit of a roadmap. I would propose setting up a shared document, e.g., hackmd, where people can share what they work on and then distill a roadmap.

> * JW: what about the github project board?

Reviewer 2: ```suggestion
* JW: You talked about deliverables — what might be useful could be a little bit of a roadmap. I would propose setting up a shared document, e.g., on hackmd, where people can share what they are working on, and then distill a roadmap.
* ...: What about the github project board?
```

* CB: We can create issue and sub-issues from more substantial comments?
* eP: Can we use the W3C CG github repo for that?
* JW: Do you want to get straight for the repo rather than the hackmd? So people can open issues (a) what do they work on and (b) proposed work items.

> * eP: Thanks everyone, 

Reviewer: ```suggestion
* eP: Thanks everyone.
```
